ts/components/conversation/MessageBody.tsx

```diff
@@ -16,28 +16,31 @@ interface Props {
   disableJumbomoji?: boolean;
   /** If set, links will be left alone instead of turned into clickable `<a>` tags. */
   disableLinks?: boolean;
-  isPublic?: boolean;
+  isGroup?: boolean;
   i18n: LocalizerType;
+  convoId: string;
 }
-
-const renderMentions: RenderTextCallbackType = ({ text, key }) => (
-  <AddMentions key={key} text={text} />
+// eslint-disable-next-line
+const renderMentions: RenderTextCallbackType = ({ text, key, convoId }) => (
```

Comment:
Curious about what eslint was complaining about here?

Reply:
I think this is out of date, removed.